dependencies/container/container.go

```diff
@@ -16,15 +17,23 @@ type Container struct {
 
 func init() {
 	containerEnv := fmt.Sprintf("%sCONTAINER", entry.DependencyPrefix)
-	if containerDeps := env.SplitEnvToList(containerEnv); len(containerDeps) > 0 {
-		for _, dep := range containerDeps {
-			entry.Register(NewContainer(dep))
+	separator := ":"
```

Comment:
@dkrzyszczyk re my previous comment about `separator` variable. I would consider making it a constant or package variable.